### PR TITLE
daily-triage email: add simplestreams

### PR DESCRIPTION
**Pull request by paride**
Changes:
 - Cleanup the workspace before each run
 - On failures email the QA engineer only
 - Add simplestreams

### CODE REVIEW

File: daily-triage/default.yaml [hunk 1]
```diff
@@ -34,8 +34,19 @@
       - build-discarder:
           num-to-keep: 30
 
+- builder:
+    name: cleanup-workspace
```
Comment (OddBloke) [inline daily-triage/default.yaml]: Why not https://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.workspace-cleanup%20(pre-build) ?

Comment (paride) [reply daily-triage/default.yaml]: The only reason is that the plugin is not installed, or at least I don't think it is, as I didn't ask for it. I'll file a RT requesting it in all our Jenkins.

Comment (paride) [reply daily-triage/default.yaml]: And I find slightly annoying that if the plugin is not installed nothing will happen: `jenkins-jobs update` will succeed and that's it.

Comment (OddBloke) [reply daily-triage/default.yaml]: That's a reasonable response; I'm happy keeping it as-is.

**Comment by paride** [comment]
I asked for the Workspace Cleanup plugin to be installed in out Jenkins.

**Pull request merged**